### DISCUSSION TITLE
Custom index html on desktop platform

### DISCUSSION THREAD

**Discussion by MathieuAndrade** [DioxusLabs/dioxus#1290] [Q&A] [answered]
Hello ! 

Can i use a custom html in desktop platform ?
I found an example that use a custom html (PWA exemple) but i not found a doc related.

Thanks fo help

**Comment by ealmloff** [comment, marked as answer]
Yes, the example is [here](https://github.com/DioxusLabs/dioxus/blob/master/examples/custom_html.rs)

**Reply by MathieuAndrade** [reply]
Oh sorry i missed this one !

It works like a charm thanks !
